src/helpers.js

```diff
@@ -37,17 +37,3 @@ exports.detable = (slots, table) => {
   }
   return [slots, bookables]
 }
```

Comment:
I don't like moving `findCollisions` out of `helpers`, it's still very much a helper :)
If it's just to give it, its own file because helpers are growing too much, we can just give them a `helper` directory.

Besides, now that the library doesn't require the web worker / eval hack anymore, we can remove helpers from `userData` and just require them where needed. But this can be for another PR.